mathics/builtin/numbers/numbertheory.py

```diff
@@ -487,7 +487,7 @@ class NextPrime(Builtin):
     }
 
     def apply(self, n, k, evaluation):
```

Comment:
Try instead of this
```
 def apply(self, n, k, evaluation):
    "NextPrime[n_, k_Integer]"
    py_k = k.to_python(n_evaluation=evaluation)
    py_n = n.to_python(n_evaluation=evaluation)
    if type(py_k) is not int or type(py_n):
        return
```
This should do the same trick, without the overhead in the pattern matching.


```